common/src/main/java/com/redhat/cloud/notifications/models/NotificationStatus.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.models;
+
+public enum NotificationStatus {
+    // The notification failed to be created for processing
+    // This could happen when sending the notification to a different processor and it was rejected before entering a processing phase
+    FAILED_CREATION,
+    // The notification is being processed
+    // A notification was sent to a different processor and we are waiting for it's output
+    PROCESSING,
+    // The notification has been in processing state for too long
+    PROCESSING_TIMEOUT,
```

Comment:
I'm not sure we need that for now. Later maybe (in another PR)?

Reply:
We could add the state later when we are ready to implement it.